### PR TITLE
Grafana-UI: Add docs for FormattedValueDisplay

### PR DESCRIPTION
**Pull request by Clarity-89**
**What this PR does / why we need it**:
Add docs for `FormattedValueDisplay` and display it in storybook.

### CODE REVIEW

File: packages/grafana-ui/src/components/FormattedValueDisplay/FormattedValueDisplay.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import { FormattedValueDisplay } from './FormattedValueDisplay';
+
+# FormattedValueDisplay
+
+Used to display a value, which also supports prefix and suffix.
```
Comment (Clarity-89) [inline packages/grafana-ui/src/components/FormattedValueDisplay/FormattedValueDisplay.mdx:6]: @ryantxu I couldn't figure proper usage and description here, maybe you have some opinions?

**Review by hugohaggmark — APPROVED** [review]
LGTM!

File: packages/grafana-ui/src/components/FormattedValueDisplay/FormattedValueDisplay.story.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+export const basic = () => {
+  return <FormattedValueDisplay value={{ text: 'Test value' }} style={{ fontSize: 12 }} />;
+};
```
Comment (hugohaggmark) [inline packages/grafana-ui/src/components/FormattedValueDisplay/FormattedValueDisplay.story.tsx:19]: This could use more examples or knobs?

Comment (Clarity-89) [reply packages/grafana-ui/src/components/FormattedValueDisplay/FormattedValueDisplay.story.tsx:19]: That' a good idea, just couldn't figure out all the use cases myself :)

**Review by dprokop — APPROVED** [review]

**Pull request merged**